playbooks/host-setup.yaml

```diff
@@ -0,0 +1,104 @@
+###############################################################################
```

Comment:
This is an extremely nice playbook. However, to a new user of our collection, it may not be apparent what exactly it is doing. I suggest a an extended introduction/documentation block at the beginning of this playbook to give a brief outline of the steps it is performing.

Reply:
Great point @asifmahmud , definitely would be good to have some in-file documentation

Reply:
I have added documentation to the host-setup playbook, thank you for your feedback